HACKING.md

```diff
@@ -37,6 +37,8 @@ that you don't have to care about indentation anymore. For more details, please
 read the [style(9) man page from FreeBSD's
 website](http://www.freebsd.org/cgi/man.cgi?query=style).
 
+For style correction install package `astyle`. Then run `make style`.
```

Comment:
👍

Reply:
Please check if I done it correctly else please ask github.

Cheers